spec/models/project_spec.rb

```diff
@@ -50,25 +51,27 @@
 
       describe "#check_edit_access #check_view_access #check_direct_view_access" do
         it "returns true for author" do
-          expect(@project.check_edit_access(@user)).to be_truthy
-          expect(@project.check_view_access(@user)).to be_truthy
-          expect(@project.check_direct_view_access(@user)).to be_truthy
+          authorize @project_policy
```

Comment:
You can't can authorize here, also this logic need to be tested in project_policy_spec, not here. Also refer other policy specs for references

Reply:
@tachyons Then how am I supposed to reference project_policy.rb? I moved the needed functions from project.rb to project_policy.rb . Now, when project_spec.rb is trying to reference the functions, it doesn't know where to go, since they have been moved. Therefore, I'm trying to reference project_policy.rb so it has access to those functions. I don't know how though.